python/mleap/sklearn/preprocessing/data.py

```diff
@@ -532,11 +533,19 @@ def __init__(self):
 
     def serialize_to_bundle(self, transformer, path, model_name):
 
-        # compile tuples of model attributes to serialize
+        if transformer.strategy == 'most_frequent' or transformer.strategy == 'constant':
+            raise ValueError(f"Imputer strategy `{transformer.strategy}` is not supported by MLeap")
+        if isinstance(transformer.missing_values, str):
+            raise ValueError("Imputer missing values of type `str` are not supported by MLeap")
+        if transformer.add_indicator:
+            raise ValueError("Imputer parameter `add_indicator` is not supported by MLeap")
```

Comment:
This feature is not on the MLeap side of things... we impute a single column and don't make any new ones

https://scikit-learn.org/stable/modules/generated/sklearn.impute.SimpleImputer.html